core/src/main/scala/com/salesforce/op/stages/impl/insights/RecordInsightsLOCO.scala

```diff
@@ -93,8 +95,20 @@ class RecordInsightsLOCO[T <: Model[T]]
     case m => toOPUnchecked(m).transformFn
   }
   private val labelDummy = RealNN(0.0)
+  private lazy val vectorMetadata = OpVectorMetadata(getInputSchema()(in1.name))
+  private lazy val histories = vectorMetadata.getColumnHistory()
+  private lazy val featureInfo = histories.map(_.toJson(false))
+
+  /**
+   * These are the name of the stages we want to perform an aggregation of the LOCO results over ferived features
+   */
+  private val smartTextClassName = classOf[SmartTextVectorizer[_]].getSimpleName
```

Comment:
Can there be a custom text vectorizer, if yes shouldn't we handle that too ?

Reply:
Unfortunately custom Text Vectorizer cannot necessary return hashes as output. This aggregation is super limited and doesn't cover all the use cases yet.

Reply:
ok, thanks.